Chapters/03-structs.qmd

```diff
@@ -726,10 +726,10 @@ While the `deinit()` method is the method used for destroying an existing instan
 
 The `init()` and `deinit()` methods are both used extensively in Zig code, and you will see both of
 them being used when we talk about allocators in @sec-allocators.
-But, as another example, let's build a simple `User` struct to represent an user of some sort of system.
+But, as another example, let's build a simple `User` struct to represent a user of some sort of system.

```

Comment:
In theory, the use of "an" here is correct, because the next word ("user") begins with a consonant sound.

Reply:
https://english.stackexchange.com/a/105117

Reply:
Let me know if you still want me to revert.

Reply:
Uhmm very interesting, I didn't knew this rule. In this case, let's then merge as is 😉